README.md

```diff
@@ -143,6 +143,31 @@ unfortunately, this will cause a `reportInvalidCast` error when this rule is ena
 
 this means that although casting between them is a common use case, `TypedDict`s and `dict`s technically do not overlap.
 
+### `reportMultipleInheritance` - ban inheriting from multiple different base classes
```

Comment:
init should be allowed on the first base class because it will always get called